tests/acceptance/helpers/sharingHelper.js

```diff
@@ -84,6 +84,7 @@ module.exports = {
    * @returns {Promise<unknown>}
    */
   assertUserHasShareWithDetails: function (user, expectedDetailsTable, filters = {}) {
+    codify.replaceInlineTable(expectedDetailsTable)
```

Comment:
helper should not know about datatable, but, I didnot want to refactor this at the time.